psi4/run_psi4.py

```diff
@@ -133,6 +133,9 @@
     data_dir = os.path.abspath(os.path.expanduser(args["psidatadir"]))
     os.environ["PSIDATADIR"] = data_dir
 
+if "@ENABLE_PCMSolver@" in ["ON", "YES", "TRUE"]:
+    sys.path.insert(1, "@PCMSolver_PYMOD@")
+
```

Comment:
Hmm, didn't actually test it except with `ON`. And you're right that it could use more case-testing. Those are the most common cmake true values. Just looked it up and should test  `1, ON, YES, TRUE, Y`, it seems. I'll look into it. But the current works for the most common and conda cases.

```
if "ON" in ["ON", "YES", "TRUE"]:
    sys.path.insert(1, "/theoryfs2/ds/cdsgroup/miniconda/envs/text35/lib/python3.5/site-packages/pcmsolver")
```